corehq/apps/callcenter/views.py

```diff
@@ -40,6 +41,7 @@ def static_options(self):
         return []
 
 
+@location_safe
```

Comment:
Ah, cool, I didn't see that.  Sounds good to remove for now.  Also, it can be location-safe if it doesn't use users.  Restricted users just need to see a limited data set, so if users aren't part of that data set, that's fine.